com.unity.multiplayer.mlapi/Tests/Runtime/Metrics/RPC/NetworkMetricsRpcTests.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using MLAPI.Metrics;
+using NUnit.Framework;
+using Unity.Multiplayer.NetworkProfiler;
+using Unity.Multiplayer.NetworkProfiler.Models;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace MLAPI.RuntimeTests.Metrics.RPC
+{
+    public class NetworkMetricsRpcTests
+    {
+        NetworkManager m_Server;
+        NetworkManager m_Client;
+        NetworkMetrics m_ClientMetrics;
+        NetworkMetrics m_ServerMetrics;
+
+        [UnitySetUp]
+        public IEnumerator SetUp()
+        {
+            if (!MultiInstanceHelpers.Create(1, out m_Server, out NetworkManager[] clients))
+            {
+                Debug.LogError("Failed to create instances");
```

Comment:
Assert will log the failure I think so we shouldn't need double logs?

Reply:
Good to know, I will look at that and remove duplicates